core/unit_test/TestTeam.hpp

```diff
@@ -970,7 +970,11 @@ struct ClassNoShmemSizeFunction {
                 double *, ExecSpace,
                 Kokkos::MemoryTraits<Kokkos::Unmanaged> >::shmem_size(1600);
 
-    int team_size = 8;
+#ifdef KOKKOS_ENABLE_SYCL
+    int team_size = 4;
+#else
+    int team_size      = 8;
```

Comment:
The formatting looks interesting.

Reply:
I know but I couldn't quite figure out why (and it didn't seem to be too important to me).